bigbluebutton-html5/imports/ui/components/nav-bar/nav-bar-graphql/talking-indicator/service.ts

```diff
@@ -3,10 +3,12 @@ import { makeCall } from '/imports/ui/services/api';
 
 const TALKING_INDICATOR_MUTE_INTERVAL = 500;
 
-export const muteUser = debounce(
+const muteUser = debounce(
   { delay: TALKING_INDICATOR_MUTE_INTERVAL },
-  (id: string, muted: boolean, isBreakout: boolean, isModerator: boolean) => {
+  (id: string, muted: boolean | undefined, isBreakout: boolean, isModerator: boolean) => {
     if (!isModerator || isBreakout || muted) return;
     makeCall('toggleVoice', id);
-  }
+  },
 );
+
+export default muteUser;
```

Comment:
Please, make it easier to add other functions and return the export to the muteUser function

Reply:
Done